vsts/determine_tests_to_run.ps1

```diff
@@ -39,6 +39,14 @@ function ShouldSkipPnPTests
 	return $false
 }
 
+function ShouldSkipDeviceStreamTests
+{
+	return ($env:SOURCE_BRANCH_NAME -and `
+		$env:SOURCE_BRANCH_NAME.toLower() -eq 'preview' -and `
+		$env:PIPELINE_ENVIRONMENT -and `
+		$env:PIPELINE_ENVIRONMENT.toLower() -eq 'prod')
```

Comment:
q - what are possible values for this env variable?

Reply:
Good idea. Done.